src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -503,11 +508,20 @@ sharded_store::get_subject_version_written_at(subject sub, schema_version ver) {
 
 ss::future<bool> sharded_store::delete_subject_version(
   subject sub, schema_version ver, force force) {
-    auto sub_shard{shard_for(sub)};
-    co_return co_await _store.invoke_on(
-      sub_shard, _smp_opts, [sub{std::move(sub)}, ver, force](store& s) {
+    auto schema = co_await get_subject_schema(sub, ver, include_deleted::yes);
```

Comment:
nitpick: A cheaper call would be `get_subject_version_id`

Reply:
Done